src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/database/OppfolgingstilfelleBitQuery.kt

```diff
@@ -111,6 +112,44 @@ fun DatabaseInterface.getUnprocessedOppfolgingstilfelleBitList() =
         }
     }
 
+const val queryGetNotReadyOppfolgingstilfelleBitList =
+    """
+    SELECT *
+    FROM TILFELLE_BIT
+    WHERE NOT ready
+    ORDER BY inntruffet ASC, id ASC
+    LIMIT 2000;
+    """
```

Comment:
Hva er egentlig flyten for de som ikke blir med gjennom `LIMIT`en her? De bare kjøres på neste cron-job?

Reply:
Ja, de blir ikke med i resultatet av spørringen, og så lenge ingen endrer databasen i mellomtiden vil de bli med i neste spørring (eller neste neste...)

Reply:
Ser at cronjob'en som allerede kjører prosesserer over 1000 biter noen ganger nå, så setter derfor limit til 2000.